app/src/components/AnalysisPane.module.css

```diff
@@ -54,7 +50,8 @@
 }
 
 .resultTable {
```

Comment:
Since the user of this class is now a chakra component, let's use style props to remove this css declaration rather than mixing style props and css.

Reply:
Will do.